client/src/models/ConfigModel.ts

```diff
@@ -3,18 +3,27 @@
 
 import { Theme } from '@fluentui/theme';
 
-export type PostCallSurveyType = 'msforms' | 'custom';
-interface MSFormsSurveyOptions {
+export type PostCallSurveyType = 'msforms' | 'custom' | 'onequestionpoll';
+export interface MSFormsSurveyOptions {
   surveyUrl: string;
 }
-interface CustomSurveyOptions {
+export interface CustomSurveyOptions {
   surveyUrl: string;
 }
 
+export type OneQuestionPollPromptType = 'likeorDislike' | 'rating' | 'text';
```

Comment:
lets name this OneQuestionPollType instead

Reply:
Agreed, done!